runc.go

```diff
@@ -655,6 +656,10 @@ func (r *Runc) args() (out []string) {
 	if r.SystemdCgroup {
 		out = append(out, "--systemd-cgroup")
 	}
+	if r.Rootless != nil {
```

Comment:
shouldn't this be `if r.Rootless != nil && *r.Rootless {`

then just add the flag.  I don't think `--rootless true` is valid, it's usually `--rootless=true`

Reply:
`*r.Rootless` is expected to be set to `false` when the caller wants to execute `runc --rootless=false` explicitly, so as to disable automatic rootless mode detection.

For example, when runc is being executed in a rootless runc, `--rootless` might need to be set to `false`explicitly. https://github.com/opencontainers/runc/issues/1837